### ISSUE TITLE
Simulate a press on Siri button on the remote ?

### ISSUE THREAD

**Issue by antro31** [postlund/pyatv#2023]
### What do you need help with?

Hello, and first of all thanks for this tremendous work. 

I was wondering if there is a way to simulate the press on the Siri button from the apple remote?
From pyatv website (Buttons/Commands (_hidC)), looks like this is possible, but I have no idea on how to do it. 

My overall idea would be to set my Apple TV Siri parameter to "Write to Siri". Once done, simulating a Siri press would open a text field to enter a request to Siri.

Would you please help on this?

**Comment by postlund** [comment]
Yeah, that should be possible I guess. I can't really test it right now, but it's easy to verify if you want to make an attempt? Just modify one of the other buttons (like "menu") in `pyatv/protocols/companion/__init__.py` to send `HidCommand.Siri` instead. Then just use `atvremote` to call the button (make sure to provide companion credentials).

**Comment by antro31** [comment]
Thanks, I actually cannot test it by now. Would anyone be so kind to test if this works? 

@postlund: if this works, do you think you could add a new function on the remote to invoke siri?

**Comment by postlund** [comment]
@antro31 If it works I can of course add it!
